pyvista/plotting/render_window_interactor.py

```diff
@@ -547,107 +547,196 @@ def enable_trackball_style(self):
         self._style_class = None
         self.update_style()
 
-    def enable_2d_style(self):
-        """Set the interactive style to 2D.
-
-        For a 3-button mouse, the left button pans, the
-        right button dollys, the middle button spins, and the wheel
-        dollys.
-        ctrl + left button spins, shift + left button dollys,
-        ctrl + middle button pans, shift + middle button dollys,
-        ctrl + right button rotates in 3D, and shift + right button
-        dollys.
+    def enable_custom_trackball_style(
+        self,
+        left="rotate",
+        shift_left="pan",
+        control_left="spin",
+        middle="pan",
+        shift_middle="pan",
+        control_middle="pan",
+        right="dolly",
+        shift_right=None,
```

Comment:
This is actually 'environment rotate', see https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6566.  All the other options have named defaults, so it is a bit weird to keep this as `None`.  If we do implement this, we can also consider getting rid of the `None` functionality to keep defaults.

Reply:
I didn't implement this yet, as I don't use this feature.  I need to investigate how it works for testing.